src/i18n/pt-BR/index.js

```diff
@@ -6,85 +6,85 @@ export default {
     message: {
       ...potato,
       help: {
-        label: 'Summon Wizard'
+        label: 'Sumonar mago'
       },
       print: { ...print },
       tester: { ...tester },
       keyboard: {
-        label: 'keyboard'
+        label: 'telcado'
       },
       layout: {
         label: 'layout'
       },
       keymapName: {
-        label: 'Keymap Name',
-        placeholder: 'custom keymap name'
+        label: 'Nome da keymap',
+        placeholder: 'nome customizavel da keymap'
       },
       loadDefault: {
-        label: 'Load Default',
-        title: 'Load default keymap from QMK Firmware'
+        label: 'Carregar padrão',
+        title: 'Carregar keymap padrão do QMK Firmware'
       },
       compile: {
-        label: 'Compile',
-        title: 'Compile Keymap'
+        label: 'Compilar',
+        title: 'Compilar Keymap'
       },
       downloadKeymap: {
-        title: 'Download keymap.c only',
-        label: 'keymap only'
+        title: 'Baixar somente keymap.c',
+        label: 'soment keymap'
       },
       downloadSource: {
         title: 'Download QMK Firmware code',
         label: 'Full Source'
       },
       downloadJSON: {
-        title: 'Export QMK Keymap JSON file',
+        title: 'Exportar arquivo JSON QMK Keymap',
```

Comment:
I would take arquivo out. Just “Exportar JSON do keymap QMK” looks like a good fit. Wdyt?

If you agree, change the other instances as well.